### DISCUSSION TITLE
Meeting Agenda 2/29

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#311] [Meeting Agendas/Minutes]
1. Review tasks
1. Market potential doc
1. Demo collector?
1. Meet with Darian
    1. turn in project abstract doc
    2. turn in weekly report

**Comment by moreno-michael** [comment]
Meeting minutes 2/29

1. Task Review
- Issue #1360 completed, Issue #1295 in progress
- Issues #1337 and #1409 in progress
- Issues #1338 and and #1339 in progress
2. Market potential doc
- Discussed and turned in 
3. Demo collector
- Issue #1427 created
- Viability of force graph discussed
- Potential additions discussed - filter, number count of entities on screen, legend (#1427)
4. Meet with Darian
- Task update
- Project abstract doc done but waiting for canvas submission to become available
- Weekly report to be updated and turned in later today
